src/shouldProvide.ts

```diff
@@ -2,7 +2,8 @@ import { State } from './State';
 
 export function shouldProvide(state: State) {
     return isImportOrRequire(state.textCurrentLine, state.cursorPosition)
-        && !startsWithADot(state.textCurrentLine, state.cursorPosition);
+        && !startsWithADot(state.textCurrentLine, state.cursorPosition)
+        && !startsWithAtPath(state.textCurrentLine, state.cursorPosition);
```

Comment:
```suggestion
        && !startsWithAnAtSymbol(state.textCurrentLine, state.cursorPosition);
```

Probably this name should be changed to clarify it's use.